### PR TITLE
[RB-15] Home page

### PR DESCRIPTION
**Pull request by liuba83**
## Description
Create Home page that contains image, search form, text bullet points with icons, books list

## Changes
Add new page

## Related issues
RB-15

## Screenshots
Screen Shot 2024-12-02 at 11 49 15 PM

### CODE REVIEW

**Review by EvgeniiRychkov — COMMENTED** [review]
Looks great, but I think more responsive design needs to be added so that the main page looks like the mockups in Figma. Also, note that six books don't fit on the screen and don't wrap to a new row.

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -1,5 +1,117 @@
...
+            type="text"
+            value={isbn}
+            onChange={handleIsbnChange}
+          >
+            ISBN
+          </Input>
+          <Button
+            as="button"
+            type="submit"
+            className="mt-7 w-full rounded-md bg-darkGreen p-2 font-semibold tracking-wide text-white"
```
Comment (EvgeniiRychkov) [inline src/pages/Home.jsx]: You can add it

`transition-transform duration-150 hover:bg-darkGreenHover active:scale-95`

**Review by VadimDmitr — COMMENTED** [review]
Great work, Liuba! It looks really good. I agree with Evgenii’s comment. Since the presentation will start with the home page, the design should be optimized for desktop and most likely starting from tablet resolution.

**Review by in43sh — APPROVED** [review]

**Review by VadimDmitr — CHANGES_REQUESTED** [review]
Looks good! But the book section seems slightly shifted to the left. For tablet and larger devices, we can align the book section like:
 ```
<h1 className="mb-8 tablet:ml-20 font-headings text-xl font-semibold md:self-start">
    Recently added books
  </h1>
  <div className="tablet:ml-20">
    <BooksList list={booksList} />
  </div>
</div>
```
Feel free to adjust margins or other styles as you see fit!

**Review by VadimDmitr — APPROVED** [review]
The page and responsiveness look awesome! Thank you for changes!

**Review by EvgeniiRychkov — APPROVED** [review]

**Review by in43sh — APPROVED** [review]

**Pull request merged**